packages/flutter/lib/src/widgets/form.dart

```diff
@@ -250,22 +251,10 @@ class FormState extends State<Form> {
 
   void _register(FormFieldState<dynamic> field) {
     _fields.add(field);
-    if (widget.autovalidateMode == AutovalidateMode.onUnfocus) {
-      field._focusNode.addListener(() => _updateField(field));
-    }
   }
 
   void _unregister(FormFieldState<dynamic> field) {
     _fields.remove(field);
-    if (widget.autovalidateMode == AutovalidateMode.onUnfocus) {
-      field._focusNode.removeListener(()=> _updateField(field));
-    }
-  }
-
-  void _updateField(FormFieldState<dynamic> field) {
-    if (!field._focusNode.hasFocus) {
-      _validate();
-    }
```

Comment:
Since the `AutovalidateMode.onUnfocus` logic was removed below, would you be able to revert this section to its original state?

Reply:
See https://github.com/flutter/flutter/pull/156708#discussion_r1811216557